resources/js/components/pages/create-poll.vue

```diff
@@ -40,6 +60,127 @@ export default {
   components: {
     selectList
   },
-  middleware: "auth"
+  middleware: "auth",
+  data() {
+    return {
+      groups: [],
+      selectedGroup: {},
+      poll: {
+        name: "",
+        users: [{ name: localStorage.getItem("username") + " (You)" }],
```

Comment:
Forgot to remove this default value. It is overridden on page load anyway. I'll remove it along with other corrections if there are any.